.drone.yml

```diff
@@ -150,8 +226,10 @@ steps:
       platforms:
         - linux/amd64
         - linux/arm64
+        - linux/s390x
       target: "rancher/machine:${DRONE_TAG}"
       template: "rancher/machine:${DRONE_TAG}-ARCH"
+      ignore_missing: true
```

Comment:
Ignoring something that is missing seems like we are ignoring a problem. Is this necessary?

Reply:
All steps in s390x pipeline https://github.com/rancher/machine/pull/154/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R153 has `failure:ignore`. So if s390x pipeline fails, the relevant docker image for s390x is not found. But the manifest pipeline will not fail due to `ignore_missing: true` thus not blocking rancher releases due to problems in s390x pipeline.....

Reply:
machine releases independently and it's process is much simpler and to me personally im comfortable dropping the failure ignore and the ignore_missing as i think this repo has a little more wiggle room than rancher/rancher

Reply:
Removed and updated the PR.